pkg/clients/rds/fake/dbsubnetgroup.go

```diff
@@ -19,17 +19,20 @@ package fake
 import (
```

Comment:
nitpick: might make sense to move this file `fake/dbsubnetgroup.go` from `clients/rds` to `clients/dbsubnetgroup`